change_notes/2024-01-25-fix-reported-fp-for-a8-4-7.md

```diff
@@ -0,0 +1,2 @@
+`A8-4-7`: ` cpp/autosar/in-parameters-for-cheap-to-copy-types-not-passed-by-value`
```

Comment:
```suggestion
`A8-4-7` - `InParametersForCheapToCopyTypesNotPassedByValue.ql`:
```

Apply formatting described in the developer handbook.

Reply:
done